spreadsheet-weather-based-marketing/classes/any-api.gs

```diff
@@ -0,0 +1,86 @@
+/**
+    Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+/**
+ * Class to do calls and process output for any API
+ */
+class AnyApi {
```

Comment:
ExternalApi, ApiWrapper,...?
AnyApi sounds clunky, but this is just my opinion

Reply:
Let's keep "AnyApi", since it represents the new approach ("not only weather"). In future we can adjust it